lib/api/multiObjectDelete.js

```diff
@@ -246,16 +374,16 @@ function multiObjectDelete(authInfo, request, log, callback) {
                     signatureAge: authParams.params.data.signatureAge,
                 },
                 parameterize: {
-                    specificResource: objects,
+                    specificResource: objects.map(entry => entry.key),
```

Comment:
if there is a version ID it should be sent so it can be checked in a policy check that includes a version id condition.